### DISCUSSION TITLE
Anyone mind if I do some code cleanup on drivers/other/blas_server_win32.c?

### DISCUSSION THREAD

**Discussion by mseminatore** [OpenMathLib/OpenBLAS#4417] [General]
The code is currently a bit messy with indent/whitespace issues, inconsistent comments, etc. I also think the monitoring is broken/incomplete.

**Comment by martin-frbg** [comment]
I'm not generally in favour of fixing indent/whitespace issues as it can make diffs harder to read, but this file has not received much attention in recent years - so I'd say go for it. Broken monitoring would be worrying indeed.
